src/components/SecondaryPanes/WhyPaused.js

```diff
@@ -17,11 +17,20 @@ class WhyPaused extends Component {
     }
 
     const message = pauseInfo.getIn(["why"]).get("message");
-    if (!message) {
-      return null;
+    if (message) {
+      return dom.div({ className: "message" }, message);
+    }
+
+    const exception = pauseInfo.getIn(["why"]).get("exception").toJS();
```

Comment:
I think you should be able to do `pauseInfo.getIn(["why", "exception"])` here.  I'm pretty sure I did the `getIn().get()` pattern above for `message` and I think I made a mistake.